SU-HSDO/suhumsci
# ARCH-54 header styles

# READY FOR REVIEW

# Summary
- Added Archaeology specific SCSS file
- Added necessary mixin; stubbed out utilities folder with two mixins
- Added sun-colored dashes to headings 1-6 in main content region for Archaeology

# Needed By (Date)
- End of sprint (5/9)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Verify that headings 1-6 have a sun-colored dash below them in the main content region

# Affected Projects or Products
- Archaeology
- suhumsci

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/ARCH-54

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
